dependencies.toml

```diff
@@ -652,6 +652,9 @@ version.ref = "kotlin-coroutine"
 [libraries.kotlin-coroutines-rx2]
 module = "org.jetbrains.kotlinx:kotlinx-coroutines-rx2"
 version.ref = "kotlin-coroutine"
+[libraries.kotlin-coroutines-test]
```

Comment:
Question) Is this library actually used? or is it possible to delete it?

Reply:
Although you don't need that library for your current work.
However, as mentioned in [this comment](https://github.com/line/armeria/pull/4724#discussion_r1130563694), it seems that the library can be used when additional test codes are needed.

Even if we don't use it, registering it in the version catalog in advance seems to be convenient for later use. I'll follow the guide of the project maintainers for this part 🙂

Reply:
Now we needed this library for [runTest](https://kotlinlang.org/api/kotlinx.coroutines/kotlinx-coroutines-test/kotlinx.coroutines.test/run-test.html) in [CoroutineServerInterceptorTest](https://github.com/line/armeria/commit/fe5b69dbecb6279b02b5c688da832e4d6852f0c1)